dot/rpc/modules/state_integration_test.go

```diff
@@ -30,27 +30,31 @@ const (
 func TestStateModule_GetRuntimeVersion(t *testing.T) {
 	// expected results based on responses from prior tests
 	expected := StateRuntimeVersionResponse{
-		SpecName:         "node",
-		ImplName:         "substrate-node",
-		AuthoringVersion: 10,
-		SpecVersion:      264,
+		SpecName:         "westend",
+		ImplName:         "parity-westend",
+		AuthoringVersion: 2,
+		SpecVersion:      9290,
```

Comment:
Where did yu get all the info regarding versions? Does it all correspond to the current westend state?

Reply:
@jimjbrettj thanks for the question, we are using westend v0.9.29 as defined [here](https://github.com/ChainSafe/gossamer/blob/fc1055dddce38cdcd61ec9afb593a10a30e2fde9/lib/runtime/constants.go#L54-L58).
So you can get the[ version data](https://github.com/paritytech/polkadot/blob/v0.9.29/runtime/westend/src/lib.rs#L105-L117) for that version looking into that release in substrate repo

Reply:
Perfect!

Reply:
Should we comment in the link to the version data above `expected := StateRuntimeVersionResponse{`? It would be a nice little addition 😉